test/integration/master/kube_apiserver_test.go

```diff
@@ -98,10 +98,15 @@ func TestRun(t *testing.T) {
 	}
 }
 
-func endpointReturnsStatusOK(client *kubernetes.Clientset, path string) bool {
-	res := client.CoreV1().RESTClient().Get().AbsPath(path).Do(context.TODO())
+func endpointReturnsStatusOK(t *testing.T, client *kubernetes.Clientset, path string) bool {
+	res := client.CoreV1().RESTClient().Get().RequestURI(path).Do(context.TODO())
 	var status int
 	res.StatusCode(&status)
+	raw, err := res.Raw()
+	if err != nil {
```

Comment:
If we're going to do this, then we should probably just fully convert this function so that it doesn't return the bool. 

```
if err != nil || status != http.StatusOK {
   t.Fatalf("got %v but wanted 200, error: %v", status, err)
}
```

Reply:
And then the calls on 121 and 124 don't have to be in conditional form.

Reply:
changed the function to return error actually